src/DraxScrollView.tsx

```diff
@@ -248,6 +249,7 @@ export const DraxScrollView = ({
 					onContentSizeChange={onContentSizeChange}
 					onScroll={onScroll}
 					scrollEventThrottle={scrollEventThrottle}
+					refreshControl={refreshControl}
```

Comment:
Which means that it should already be passed to the ScrollView as part of the spread `{...props}` here. Is this not already working for you?